docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-azure-ad.md

```diff
@@ -221,6 +221,18 @@ To complete configuration, enter information about your AD instance in the Ranch
 
     <code>http<span>s://g</span>raph.microsoft.com<del>/abb5adde-bee8-4821-8b03-e63efdc7701c</del></code>
 
+1. (Optional) In Rancher v2.9.0 and later, you can filter user group memberships in Azure AD to reduce the amount of log data generated. 
+    
+    :::warning
+    
+    Filtering out a user group membership can have the side effect of denying users permissions they should have.
+
+    :::
+    
+    1. Click the checkbox next to **Limit users by group membership**.
```

Comment:
This seems repetitive provided that its already defined well in step 329. Can we remove this part and only have the part added in lines 329 through 349?